flowSteps/listMembersInfoMailchimp/step.js

```diff
@@ -0,0 +1,13 @@
+step.listMembersInfoMailchimp = function (inputs) {
+
+	var inputsLogic = {
+		listId: inputs.listId || "",
+	};
+
+	var options = {
+		path: "/lists/" + inputsLogic.listId + "/members"
+	}
+
+	return endpoint._get(options);
```

Comment:
Although we still don't have support for flowSteps we should migrate the references of these, previously they said for example `endpoint._get(...` now the call would be something like: `pkg.mailchimp.get(...` it should be only one call for each javascript file of steps.